src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -396,13 +396,13 @@ private function getClasses(array $entrypointProperties, array $classes): array
                 [
                     '@type' => 'hydra:SupportedProperty',
                     'hydra:property' => [
-                        '@id' => '#ConstraintViolationList/violation',
-                        '@type' => 'rdf:Property',
-                        'rdfs:label' => 'violation',
+                        '@id' => '#ConstraintViolationList/violations',
+                        '@type' => 'hydra:Link',
```

Comment:
Using `Link` when the resource is embedded is fine. The embedded node can be used to warm up a cache for instance. As it is a non-authoritative optimization the client might choose to ignore though.